apps/prairielearn/src/pages/instructorAssessmentManualGrading/assessmentQuestion/utils/columnDefinitions.tsx

```diff
@@ -0,0 +1,420 @@
+        if (enrollmentId) {
+          return <a href={getStudentEnrollmentUrl(urlPrefix, enrollmentId)}>{uid}</a>;
+        }
+        return uid;
+      },
+      enableHiding: true,
+    }),
+
+    // Grading status column
+    columnHelper.accessor('requires_manual_grading', {
+      id: 'requires_manual_grading',
+      header: 'Grading status',
+      cell: (info) => (info.getValue() ? 'Requires grading' : 'Graded'),
+      filterFn: ({ getValue }, columnId, filterValues: string[]) => {
+        if (filterValues.length === 0) return true;
+        const requiresGrading = getValue(columnId);
+        const status = requiresGrading ? 'Requires grading' : 'Graded';
+        return filterValues.includes(status);
+      },
+    }),
+
+    // Assigned grader column
+    columnHelper.accessor('assigned_grader_name', {
+      id: 'assigned_grader_name',
+      header: 'Assigned grader',
+      cell: (info) => info.getValue() || 'Unassigned',
+      filterFn: (row, columnId, filterValues: string[]) => {
+        if (filterValues.length === 0) return true;
+        const current = row.getValue<InstanceQuestionRow['assigned_grader_name']>(columnId);
+        if (!current) return filterValues.includes('Unassigned');
+        return filterValues.includes(current);
+      },
+    }),
+
+    // Auto points column (only if assessment has auto points)
+    ...(assessmentQuestion.max_auto_points && assessmentQuestion.max_auto_points > 0
+      ? [
+          columnHelper.accessor('auto_points', {
+            id: 'auto_points',
+            header: 'Auto points',
+            cell: (info) =>
+              formatPointsWithEdit({
+                row: info.row.original,
+                field: 'auto_points',
+                hasCourseInstancePermissionEdit: authzDataHasCourseInstancePermissionEdit,
+                urlPrefix,
+                csrfToken,
+              }),
+            filterFn: numericColumnFilterFn,
+          }),
+        ]
+      : []),
+
+    // Manual points column
+    columnHelper.accessor('manual_points', {
+      id: 'manual_points',
+      header: 'Manual points',
+      cell: (info) =>
+        formatPointsWithEdit({
+          row: info.row.original,
+          field: 'manual_points',
+          hasCourseInstancePermissionEdit: authzDataHasCourseInstancePermissionEdit,
+          urlPrefix,
+          csrfToken,
+        }),
+      filterFn: numericColumnFilterFn,
+    }),
+
+    // Total points column (hidden by default)
+    columnHelper.accessor('points', {
+      id: 'points',
+      header: 'Total points',
+      cell: (info) =>
+        formatPointsWithEdit({
+          row: info.row.original,
+          field: 'points',
+          hasCourseInstancePermissionEdit: authzDataHasCourseInstancePermissionEdit,
+          urlPrefix,
+          csrfToken,
+        }),
+      filterFn: numericColumnFilterFn,
+      enableHiding: true,
+    }),
+
+    // Score percentage column (not in AI grading mode)
+    ...(!aiGradingMode
```

Comment:
Why make this completely unavailable in AI grading mode? Our strategy has been for AI grading to change the default set of columns, but not to have it prevent the user from toggling on certain columns.

For completeness: of course there are columns that are _only_ visible in AI grading mode, I'm talking specifically about columns that are also visible in the non-AI-grading view.